core/src/main/kotlin/DokkaGenerator.kt

```diff
@@ -152,7 +145,7 @@ class DokkaGenerator(
 
     private fun createEnvironmentAndFacade(
         configuration: DokkaConfiguration,
-        pass: DokkaConfiguration.PassConfiguration
+        pass: DokkaSourceSet
```

Comment:
Should we stick with this concept of naming something "pass" 🤔

Reply:
Changed